### PR TITLE
[service.subtitles.titlovi-com-official] 1.1.0

### PR DESCRIPTION
**Pull request by tturkalj**
### Description

Added Croatian translation, updated addon description
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
You didn't pull latest changes from our repo before doing another PR.
Please read http://kodi.wiki/view/Git_usage

File: service.subtitles.titlovi-com-official/addon.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+       provider-name="titlovi.com">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.requests" version="2.18.4"/>
+        <import addon="script.module.simplecache" version="1.0.11"/>
+    </requires>
+    <extension point="xbmc.subtitle.module" library="service.py"/>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Titlovi.com</summary>
```
Comment (romanvm) [inline service.subtitles.titlovi-com-official/addon.xml]: Please use 4-letter language codes, e.g. `en_GB`.

**Comment by Lunatixz** [comment]
After requested changes LGTM

File: service.subtitles.titlovi-com-official/addon.xml [hunk 2]
```diff
@@ -12,12 +12,17 @@
     <extension point="xbmc.addon.metadata">
         <platform>all</platform>
         <summary lang="en">Titlovi.com</summary>
-        <description lang="en">Search and download Titlovi.com subtitles. Click here to read more info about manual searching.</description>
-        <disclaimer>Users need to provide Titlovi.com username and password in add-on configuration</disclaimer>
+        <description lang="en">Search and download [B]Titlovi.com[/B] subtitles[CR][CR]When using manual search for [B]TV shows[/B] please use this example format for season and episode numbers:[CR]The Wire [B]S01E01[/B]</description>
+        <description lang="hr">Pretražite i preuzmite podnaslove sa [B]Titlovi.com[/B].[CR][CR]Kada koristite prilagođenu pretragu za [B]TV serije[/B] koristite format za sezonu i epizodu kao u ovom primjeru:[CR]The Wire [B]S01E01[/B]</description>
+        <disclaimer>Users need to provide [B]Titlovi.com[/B] username and password in add-on configuration</disclaimer>
+        <disclaimer lang="hr">Korisnici moraju upisati [B]Titlovi.com[/B] korisničko ime i lozinku u postavke dodatka</disclaimer>
```
Comment (Rechi) [inline service.subtitles.titlovi-com-official/addon.xml]: @Lunatixz requested changes are still pending

**Comment by tturkalj** [comment]
@romanvm I have fixed language codes in addon.xml

**Pull request merged**